sdk/go/auto/optup/optup.go

```diff
@@ -193,6 +193,8 @@ type Options struct {
 	SuppressProgress bool
 	// Suppress display of stack outputs (in case they contain sensitive values)
 	SuppressOutputs bool
+	// ContinueOnError will continue updating resources even if a resource update fails
```

Comment:
Why did this get different help text?

Reply:
I probably just trusted copilot a bit too much :grimacing: Made it the same as the others now.